import-service/src/lambda/import-file-parser/import-file-parser.ts

```diff
@@ -1,14 +1,40 @@
 import { S3Handler } from 'aws-lambda';
-import { S3 } from 'aws-sdk';
+import { S3, SQS } from 'aws-sdk';
 import { GetObjectRequest } from 'aws-sdk/clients/s3';
 import * as csvParser from 'csv-parser';
+import { Transform } from 'stream';
 import { BUCKET_NAME, DEFAULT_REGION } from '../../utils/configs';
 
+class SQSStream extends Transform {
```

Comment:
`Transform` stream is more suitable if you need to pass through data. 
In this case, `Writable` would suit better.